Formula/juliaup.rb

```diff
@@ -17,8 +17,13 @@ class Juliaup < Formula
 
   depends_on "rust" => :build
 
+  conflicts_with "julia", because: "both install `julia` binaries"
+
   def install
-    system "cargo", "install", *std_cargo_args
+    system "cargo", "install", "--bin", "juliaup", *std_cargo_args
+    system "cargo", "install", "--bin", "julialauncher", *std_cargo_args
```

Comment:
How does doing two `cargo install`s differ from doing just one?

Reply:
Ah, I see. This omits `juliainstaller`.

Reply:
Yeah, I had hoped that `--bin juliaup,juliainstaller` works, but it doesn't...